### ISSUE TITLE
Coins in the form of objects

### ISSUE THREAD

**Issue by ZConnorZ** [LemmoTresto/LemonMobCoins#27]
You can add coins as an object that will drop out of the creature if it died. 

Falls even if the creature did not die at the hands of the player.

![InConfigName](https://user-images.githubusercontent.com/45974097/65962595-b91a9b00-e461-11e9-89a7-52a722205f09.png)

**Comment by LemmoTresto** [comment]
Good idea, this will be implemented for optional use. In combination with withdrawing coins.

**Comment by ZConnorZ** [comment]
You can also make the coins taken from the chest or funnel automatically issued to the player?
